.github/workflows/memcheck.yml

```diff
@@ -241,13 +241,25 @@ jobs:
           fi
         if: always()
 
-      - name: Tar log files
+      - name: Tar node log files
         shell: bash
         continue-on-error: true
         run: |
           find $HEAPNODE_DATA_PATH -iname '*.log*' | tar -zcvf heap_node_log_files.tar.gz --files-from -
           find $NODE_DATA_PATH -iname '*.log*' | tar -zcvf nodes_log_files.tar.gz --files-from -
+        if: failure()
+
+      - name: Tar client log files
+        shell: bash
+        continue-on-error: true
+        run: |
```

Comment:
I think the reason for us not getting the client logs is that the `$CLIENT_LOG_PATH` has changed and as well as the option to enable client logs. We need to pass the following arg to enable it `--log-output-dest data-dir`. I will work on this and raise a PR for it.